src/app.js

```diff
@@ -52,7 +53,9 @@ adminApp.use(baseEndpoint, adminAppRouter);
 
 appRouter.use(loggerMiddleware);
 appRouter.use(authentication);
+appRouter.use(validateBooleanParams);
 adminAppRouter.use(authentication);
+adminAppRouter.use(validateBooleanParams);
```

Comment:
Why does adminAppRouter need this middleware?

Reply:
What is adminAppRouter for?

Reply:
[This](https://github.com/osu-mist/express-api-skeleton/blob/92b756a8f169fdd067f64dbd4aed52348b77972b/src/app.js#L82-L99) and it's mainly for health check.